components/neeva/renderer/content_filtering_agent.cc

```diff
@@ -133,12 +133,7 @@ ContentFilteringPolicy ContentFilteringAgent::GetPolicyForRequest(
     return ContentFilteringPolicy::kAllow;
   }
 
-  // There's no point in blocking foo.com from making requests from first-party origins (e.g. foo.com). 
-  // Doing so can break logins or other site functionality. 
-  if (!IsThirdParty(url, first_party_origin)) {
-    // Downgrade from kBlockRequests to kBlockCookies to minimize impact on sites. 
-    return ContentFilteringPolicy::kBlockCookies;
-  }
+  bool isUrlThirdParty = IsThirdParty(url, first_party_origin);
```

Comment:
nit: chromium c++ style uses unix_style naming convention instead of camelCase for variables.